hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -1594,6 +1598,107 @@ public void createVolume(OmVolumeArgs args) throws IOException {
     }
   }
 
+  @Override
+  public VolumeList startCreateVolume(OmVolumeArgs args) throws IOException {
+    try {
+      // TODO: Need to add metrics and Audit log for HA requests
```

Comment:
Let's file a follow up jira. Audit especially is critical to fix.

Reply:
Filed https://issues.apache.org/jira/browse/HDDS-1392